advocacy_docs/edb-postgres-ai/ai-accelerator/models/supported-models/nim_clip.mdx

```diff
@@ -37,18 +37,18 @@ SELECT aidb.create_model(
 );
```

Comment:
Should this topic be named such that you can tell the difference between this and the topic that simply CLIP?